modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -49,6 +50,10 @@ class ImagingUploader extends React.Component {
     this.setState({filter});
   }
 
+  resetFilters() {
+    this.refs.imagingUploaderFilter.clearFilter();
```

Comment:
if using the ref callback method used below, you should remove the `.refs.` here